From 1710e037aae8b793fd1e1e09daf50c8827e509b2 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Mon, 24 Mar 2025 13:48:46 +0000
Subject: [PATCH] Fix build with gcc-15

:Release Notes:
Add missing include.

:Detailed Notes:
Fixes:

http://gecko.lge.com:8000/Errors/Details/1142324

TOPDIR/BUILD/work/raspberrypi4_64-webos-linux/libedgetpu/grouper/git/makefile_build/../driver/usb/usb_device_interface.h:131:5: error: 'uint8_t' does not name a type
  131 |     uint8_t request_type;
      |     ^~~~~~~

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRR-5443] Create GPVB with Yocto 5.2 Walnascar
---
Upstream-Status: Pending

 driver/usb/usb_device_interface.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/driver/usb/usb_device_interface.h b/driver/usb/usb_device_interface.h
index 238bca9..54f2f6e 100644
--- a/driver/usb/usb_device_interface.h
+++ b/driver/usb/usb_device_interface.h
@@ -15,6 +15,8 @@
 #ifndef DARWINN_DRIVER_USB_USB_DEVICE_INTERFACE_H_
 #define DARWINN_DRIVER_USB_USB_DEVICE_INTERFACE_H_
 
+#include <cstdint>
+
 #include "port/array_slice.h"
 #include "port/integral_types.h"
 #include "port/status.h"
